lib/ja_serializer/serializer.ex

```diff
@@ -77,51 +77,54 @@ defmodule JaSerializer.Serializer do
 
       def type, do: fn(model, _conn) -> model.type end
   """
+  # TODO: Can we convert this to type/2 for consistency without too much hassle?
   defcallback type() :: String.t | fun()
 
   @doc """
-  Returns a map of attributes to be mapped.
+  Returns a map of attributes to be included.
```

Comment:
I wonder if "included" should be "serialized" so it's not confused with the other notion of "included" in the spec.